test/tic_tac_toe_test.exs

```diff
@@ -29,5 +33,15 @@ defmodule TicTacToeTest do
         7 | 8 | 9
         """
     end
+
+    test "(unit test) displays user input message" do
+      start_supervised!(CommunicatorMock)
+
+      TicTacToe.start(CommunicatorMock, CommandLineFormatter)
+      user_input = "1"
+
+      assert :sys.get_state(CommunicatorMockServer) =~
```

Comment:
Where is CommunicatorMockServer coming from?

Reply:
This is the name for the `GenServer` that is being used in the `CommunicatorMock`, the test is using the name as a reference to access and check the state to verify the correct changes made when the mocks functions (`display` and `read_input`) are called

Reply:
I see. So if I use a `FooBarMock` with GenServer, it's going to give me a `FooBarMockServer`?

Reply:
The `CommunicatorMockServer` name would have to be changed in the `communicator_mock` module manually as of now as it does not use the name of the module to create this server name, but I can make it do so if that makes the most sense 😄

Reply:
No worries, I was just trying to understand how GenServer works in this case. It's all good!